tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/SearchConfigSpec.scala

```diff
@@ -0,0 +1,596 @@
+package ch.epfl.bluebrain.nexus.tests.kg
+
+import akka.http.scaladsl.model.StatusCodes
+import cats.implicits._
+import ch.epfl.bluebrain.nexus.tests.BaseSpec
+import ch.epfl.bluebrain.nexus.tests.Identity.resources.Rick
+import ch.epfl.bluebrain.nexus.tests.iam.types.Permission.{Organizations, Resources}
+import io.circe.Json
+import monix.bio.Task
+import org.scalatest.Assertion
+
+class SearchConfigSpec extends BaseSpec {
+
+  private val orgId    = genId()
+  private val projId1  = genId()
+  private val id1      = s"$orgId/$projId1"
+  private val projects = List(id1)
+
+  private val neuronMorphologyId =
+    "https://bbp.epfl.ch/neurosciencegraph/data/neuronmorphologies/da3b1e42-5f7f-4065-8be3-d2132c219bc2"
```

Comment:
We can use explicit ids instead of ones with uuids
And make them a little shorter